docs/DeveloperGuide.md

```diff
@@ -259,12 +259,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a …​ | I want to …​ | So that I can…​ |
 | -------- | ---------- | --------------- | ------------------ |
-| `* * *`  | new user | see usage instructions | refer to instructions when I forget how to use the App |
-| `* * *`  | user | add a new deadline | know when to complete it by |
+| `* * *`  | new user | see all possible commands | I know what I can do in the app |
```

Comment:
Not all commands will be inside. Unless we want to change the implementation

Reply:
This is for list command, list all tasks the user has

Reply:
Refer to line 262

Reply:
oh that is the help command implementation